target/linux/brcmiproc/ps225/config-default

```diff
@@ -0,0 +1,185 @@
+CONFIG_64BIT=y
```

Comment:
Please refresh this file with
```
make kernel_oldconfig CONFIG_TARGET=subtarget V=99
```

Reply:
Refreshed using make kernel_oldconfig CONFIG_TARGET=subtarget V=99 as suggested.